requirements.txt

```diff
@@ -0,0 +1,2 @@
+renoir==1.5.4
```

Comment:
I never heard of that one before, why that one over something more famous such as Jinja?

Reply:
I'm using it for a while and these are my personal PRO / CONS about it:

**PRO: **
- It doesn't have 2 different types of delimiters, instead, the `=` is used to output the value (Eg. `[[ a  = range(1,10) ]]` ... `[[ =a ]]`).
- Reduced complexity in the codebase.
- Is quite similar to `Jinja`, but looks cleaner during code reviews (at least to me 😂).
- The author is quite involved in the Python community and other web development projects.

**CONS:**
- Is not famous as Jinja
- May not have all the features of Jinja

Reply:
Thanks for sharing.
Yeah I tend to go with "boring software", but this one is not a huge change over Jinja.
I don't think it should hurt too switch to something else if it ever goes unmaintained.
While it's probably not what I would have chosen, I have never used it andI don't have strong opinion against it.
I'm happy you took the initiative to revamp the website tech/approach